### PR TITLE
Updated to support Swift 3.

### PR DESCRIPTION
**Pull request by MadsBogeskov**

### CODE REVIEW

**Comment by MadsBogeskov** [comment]
afaik the reason Travis fails is that this should be compiled with the new Swift compiler.

File: Demo/RxSwiftExtDemo/RxSwiftExtPlayground.playground/SupportCode.remap [hunk 1]
```diff
@@ -0,0 +1,53 @@
+[
+ {
+  "file": "/Users/dkmadboe/Developer/Github/RxSwiftExt2/Demo/RxSwiftExtDemo/RxSwiftExtPlayground.playground/Sources/SupportCode.swift",
```
Comment (fpillet) [inline Demo/RxSwiftExtDemo/RxSwiftExtPlayground.playground/SupportCode.remap]: There shouldn't be absolute paths in your playground -- unless this file should not be committed?

**Comment by fpillet** [comment]
I'm not worried about the CI issues at this stage - I just need to move current to a swift-2 branch before merging your commit (will give you write access to the repo as well)

Sorry about the delay responding - I've been quite busy this week

File: Demo/RxSwiftExtDemo/RxSwiftExtPlayground.playground/SupportCode.remap (hunk 1 above)
Comment (MadsBogeskov) [inline Demo/RxSwiftExtDemo/RxSwiftExtPlayground.playground/SupportCode.remap]: Good catch. Consider it gone in 3... 2...

**Comment by MadsBogeskov** [comment]
@fpillet no problem. We have just migrated to Swift 3 at the office, so it would be great to have this in, instead of using my own fork :)

**Comment by fpillet** [comment]
I have merged this PR in the swift3 branch as of commit ccbc4e8c3281dae593dbd1405b542afbaa4044b5

Thanks! Let me know if referencing this branch from your code works for you. And thanks a lot for taking care of the swift3 transition!

**Pull request closed without merging**